libs/store-harvester/src/state/types.ts

```
@@ -0,0 +1,4 @@
+import type { toDb } from "./transform";
```

ESLint

[error] Resolve error: webpack with invalid interface loaded as resolver

(import/no-cycle)